src/cgl.rs

```diff
@@ -103,6 +137,7 @@ pub const CORE_INTEGER_ATTRIBUTES: &'static [CGLPixelFormatAttribute] =
       kCGLPFAOpenGLProfile,
       kCGLPFAVirtualScreenCount];
 
+#[link(name = "OpenGL", kind = "framework")]
```

Comment:
@pcwalton Deferring to you since I don't know much about mac. Should the opengl framework link line be added here to the cgl bindings?